ethereum/referencetests/src/main/java/org/hyperledger/besu/ethereum/referencetests/BonsaiReferenceTestWorldState.java

```diff
@@ -75,6 +75,14 @@ public ReferenceTestWorldState copy() {
         layerCopy, cachedMerkleTrieLoader, trieLogManager, preImageProxy);
   }
 
+  @Override
+  protected void verifyWorldStateRoot(final Hash calculatedStateRoot, final BlockHeader header) {
+    // if the supplied header has a non-zero state root, verify.  Else bypass
```

Comment:
nit: would be nice to say why we bypass here

Reply:
added a verbose javadoc 👍